libavfilter/f_ebur128.c

```diff
@@ -487,11 +487,11 @@ static int config_audio_output(AVFilterLink *outlink)
             !ebur128->true_peaks_per_frame || !ebur128->swr_ctx)
             return AVERROR(ENOMEM);
 
```

Comment:
FFMPEG bug from their new ch_layout stuff... I really should PR this back to ffmpeg